evaluation.md

```diff
@@ -5,15 +5,30 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+The first change we made was the addition of some parentheses to our statements. This was necessary because we couldn't figure out how to tell scala to give one operator higher precedence than another, so it was simply looking at things from left-to-right which was not always what we wanted.
+
```

Comment:
Yeah, I don't think there's any way around having some parenthesis.  My group foresaw this and decided to incorporate the parens in our initial design as a way of getting Picobot users familiar with function calls.